buidler.config.js

```diff
@@ -0,0 +1,32 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin('@nomiclabs/buidler-truffle5')
+usePlugin('@nomiclabs/buidler-web3')
```

Comment:
buidler-aragon already declares the usage of these, so it's redundant to declare them here as well.